src/System.Drawing.Common/src/Configurations.props

```diff
@@ -4,6 +4,7 @@
     <BuildConfigurations>
       netcoreapp2.0-Windows_NT;
       netcoreapp2.0-Unix;
```

Comment:
I've mentioned it over in the tracking issue for this: https://github.com/dotnet/corefx/issues/20740. I will address that before I close the overall packaging issue.